utils/utils.go

```diff
@@ -2,11 +2,15 @@ package utils
 
 import (
 	"os"
+	"path/filepath"
 
 	"github.com/dustin/go-humanize"
+	"github.com/gabriel-vasile/mimetype"
 )
 
 const MinimalOverhead = 384
```

Comment:
This value is present in the capabilities endpoint of the API. Next version?

Reply:
maybe